cmd/main.go

```diff
@@ -1,5 +1,8 @@
 package main
 
+import _ "net/http/pprof"
+import _ "github.com/grafana/pyroscope-go/godeltaprof/http/pprof"
```

Comment:
We don't need these imports. Profiling is already enabled using a middleware

Reply:
corrected by https://github.com/grafana/quickpizza/pull/34/commits/007326344263e042ae40a2a76dbf935dff6d1f1d

Thanks!